src/apiMocks/mockData.ts

```diff
@@ -43,17 +42,20 @@ export type RandomScanOptions = Partial<{
  * Generates scan result with randomly-generated mock data.
  *
  * @param options
- * @returns {ScanResult} - A single scan result.
+ * @returns A single scan result.
  */
-export function createRandomScan(options: RandomScanOptions = {}): ScanResult {
+export function createRandomScan(
+  options: RandomScanOptions = {}
+): OnerepScanResultRow {
   faker.seed(options.fakerSeed);
   return {
     id: faker.number.int(),
-    profile_id: faker.number.int(),
+    onerep_scan_result_id: faker.number.int(),
+    onerep_scan_id: faker.number.int(),
     first_name: faker.person.firstName(),
     last_name: faker.person.lastName(),
     middle_name: faker.person.middleName(),
-    age: faker.number.int({ min: 14, max: 120 }).toString(),
+    age: faker.number.int({ min: 14, max: 120 }),
```

Comment:
don't we have a check for `=="string"` for this?

Reply:
The OneRep API returns a string, which we check for, and then parse it into an integer to store it in our database. Since this mocks the data returned by the DB, it returns an integer. (Although now that you mention it, maybe there's a reason OneRep kept it a string? In which case, maybe we should keep it a string in our DB as well?)

Reply:
Oh true, long day of PR reviewing 😓